templates/devfile.yaml

```diff
@@ -0,0 +1,83 @@
+schemaVersion: 2.0.0
+metadata:
+  name: java-openliberty
+  version: 0.4.1
```

Comment:
hmm... what to name this... java-openliberty-gradle ?    So it's permanently a second-class citized behind java-openliberty (maven implied)?   

As before.. this could be 0.1